docs/fundamentals/accessibility.md

```diff
@@ -0,0 +1,378 @@
+
+> [!WARNING]
+> The `SemanticProperties.Hint` property conflicts with the `Entry.Placeholder` property on Android, which both map to the same native property. Therefore, setting a different `SemanticProperties.Hint` value to the `Entry.Placeholder` value isn't recommended.
+
+<!-- ^ and Editor? -->
+
+### Heading levels
+
+The `SemanticProperties.HeadingLevel` attached property enables an element to be marked as a heading to organize the UI and make it easier to navigate. Some screen readers enable users to quickly jump between headings.
+
+Headings have a level from 1 to 9, and are represented by the `SemanticHeadingLevel` enumeration, which defines `None`, and `Level1` through to `Level9` members.
+
+> [!IMPORTANT]
+> While Windows offers 9 levels of headings, Android and iOS only offer a single heading. Therefore, when `SemanticProperties.HeadingLevel` is set on Windows it maps to the correct heading level. However, when set on Android and iOS it maps to a single heading level.
+
+The following example demonstrates setting this attached property:
+
+```xaml
+<StackLayout SemanticProperties.Description="Main page layout"
+             SemanticProperties.HeadingLevel="Level1">
+    ...
+</StackLayout>
+```
+
+Alternatively, it can be set in C#:
+
+```csharp
+StackLayout stackLayout = new StackLayout();
+SemanticProperties.SetDescription(stackLayout, "Main page layout.");
+SemanticProperties.SetHeadingLevel(stackLayout, SemanticHeadingLevel.Level1);
+```
+
+In addition, the `SetValue` method can also be used to set the `SemanticProperties.HeadingLevel` attached property:
+
+```csharp
+stackLayout.SetValue(SemanticProperties.HeadingLevelProperty, SemanticHeadingLevel.Level1);
+```
+
+<!-- Todo: is this right?
+### WebView    
+
+The native WebView control on each platform is responsible for determining which elements of its content are accessible, and each WebView control should report its HTML content as accessibility elements.
+
+Alternatively, if the content in your `WebView` is simple, you could provide a description of the content for the `WebView` with the `SemanticProperties.Description` and `SemanticProperties.Hint` attached properties.
+
+-->
+
+## Semantic focus
+
+Controls have a `SetSemanticFocus` extension method, defined in the `Microsoft.Maui` namespace, which forces screen reader focus to a specified element. For example, given a `Label` named `label`, screen reader focus can be forced to the element with the following code:
+
+```csharp
+using Microsoft.Maui;
+...
+
+label.SetSemanticFocus();
+```
+
+<!-- Todo: Any scenarios for calling UpdateSemantics or UpdateSemanticNodeInfo? -->
+
+## Semantic screen reader
+
+.NET MAUI Essentials includes a `SemanticScreenReader` class that enables you to instruct a screen reader to announce specified text. <!-- For more information, see [SemanticScreenReader]().-->
```

Comment:
I think adding a couple sentences about the `Announce` method itself would be nice! Especially if we can note how it's different from `Description` since it's a method and not a property tied to a control